site/reference/live/migrate/README.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "`migrate`"
+linkTitle: "migrate"
+type: docs
+description: >
+Migrate a package and the inventory object to use the ResourceGroup CRD.
+---
+<!--mdtogo:Short
+    Migrate a package and the inventory object to use the ResourceGroup CRD.
+-->
+
+`migrate` moves the inventory list in the cluster from a ConfigMap resource to a 
+ResourceGroup CR and moves the inventory information on the local filesystem from
+the inventory template into the Kptfile.
```

Comment:
may want to give a back story that previous versions of kpt used CM

Reply:
Added a short section with some more background. I assume we will publish a migration guide for kpt v1 and that should probably include some more in-depth explanation of migration. I don't think we want to push too much of that information into the reference docs.